src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -1326,7 +1402,8 @@
    26 structured-facts-deferrable-constraints
    27 switch-value-string-index-to-gin
    28 lift-fact-paths-into-facts
-   29 version-2yz-to-300-migration})
+   29 version-2yz-to-300-migration
+   30 coalesce-fact-values})
```

Comment:
I'm curious about your thoughts now @wkalt, I'm resurrecting the beaten dead horse. What about merging this migration with the previous one? We've seen some problems in the 2.3.z series with migrations taking a long time and we've seen here that migrations are indeed a big performance concern.